src/frontend/js/components/DashBoard/Navigator/index.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
In order to keep folder structure as flat as possible. I think we could rename this file `Navigator.spec.tsx` then move it at the root of `js/components/Dashboard`.